platform/android/app/src/main/kotlin/coop/polypoly/polypod/Authentication.kt

```diff
@@ -64,17 +103,24 @@ class Authentication {
 
     private class PolyAuthCallback(
         val context: Context,
+        val reAuth: Boolean = false,
```

Comment:
Since this is used only to set the title, and that title is probably known by whoever is calling, can you consider the alternative of using that title as an argument? That way you're not duplicating the code that checks for reAuth and sets the title accordingly.

Reply:
mmh I get you...but then doesn't look strange that a `PolyAuthCallback` needs to know the title of a prompt ?!

Reply:
Not really...

Reply:
checked it again -  We don't call `PolyAuthCallback` from two distinct points but only from `authenticate`. if we pass the title to the function call...then it's the `authenticate` function that requires an `if-else`, so it's either-way I think. And `authenticate` is already lengthy one with other if-elses

Reply:
`authenticate` already computes `title` in line 76, before the call to `PolyAuthCallback`. What do you mean with requiring an `if-else`?